ledger/ledger-api-auth/src/main/scala/com/digitalasset/ledger/api/auth/Authorizer.scala

```diff
@@ -213,7 +213,11 @@ final class Authorizer(
   private def authorizationErrorAsGrpc[T](
       errOrV: Either[AuthorizationError, T]
   ): Either[StatusRuntimeException, T] =
-    errOrV.fold(err => Left(ErrorFactories.permissionDenied(err.reason)), Right(_))
+    errOrV.fold(
+      err =>
+        Left(LedgerApiErrors.AuthorizationChecks.PermissionDenied.Reject(err.reason).asGrpcError),
+      Right(_),
```

Comment:
I am wondering if `.left.map(err => LedgerApiErrors.AuthorizationChecks.PermissionDenied.Reject(err.reason).asGrpcError)` could work here

Reply:
I guess it would work. But merging without really checking it because it's a minor change not directly related to the PR's goal